<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QScrollBar" name="timeBar">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>380</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="pageStep">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="timeLabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>380</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>timeLabel</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>371</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>503</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu0000">
    <property name="title">
     <string>0000</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionaa"/>
    <addaction name="action52"/>
    <addaction name="action5459"/>
   </widget>
   <addaction name="menu0000"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionaa"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionaa">
   <property name="text">
    <string>aa</string>
   </property>
  </action>
  <action name="action52">
   <property name="text">
    <string>52</string>
   </property>
  </action>
  <action name="action5459">
   <property name="text">
    <string>5459</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
